tests/components/MapboxMap.stories.tsx

```diff
@@ -20,7 +21,13 @@ const meta: ComponentMeta<typeof MapboxMap> = {
       control: false,
     },
   },
-  parameters: { layout: 'fullscreen', percy: { enableJavascript: true } },
+  parameters: {
+    layout: 'fullscreen',
+    percy: {
+      enableJavascript: true,
+      waitForSelector: '.mapboxgl-map[style=\'visibility: visible;\']'
```

Comment:
nice!

Reply:
decided to use the storybook play function to force await for the dynamic map instead

Reply:
made an item for it: https://yexttest.atlassian.net/browse/SLAP-2462